_includes/video-player-embed.html

```diff
@@ -28,21 +28,19 @@ <h2>Select source</h2>
         </div>
       </div>
     </div>
-    <div id="LoadingStream" class="selector-option" style="display:none;">
+    <div id="loadingStream" class="selector-option" style="display:none;">
       {% include graphics/video/loader-animation.svg %}
       <h3 id="loadingTitle" class="mt-0">Locating stream...</h3>
       <div class="stream-message" id="msg"></div>
     </div>
   </div>
 
-  <div class="share-container w-100 mt-1">
-    <div class="flex mb-1">
-      <button class="share-link compact">Share link</button>
-      <button class="share-tweet ml-1 compact">Share on Twitter</button>
-    </div>
-    <input type="text" class="input none compact" id="link" />
+  <div class="share-container w-100 mt-1 flex">
+    <button class="share-link mr-1 compact">Share link</button>
+    <button class="share-tweet mr-1 compact">Share on Twitter</button>
+    <input type="text" class="compact" style="flex:1; text-overflow: ellipsis" id="link" />
```

Comment:
@garrying sorry I overwrote your changes, but I want the field to show a continually updating hash, can we make it look like this?